consensus/state.go

```diff
@@ -1295,6 +1295,37 @@ func (cs *State) defaultDoPrevote(height int64, round int32) {
 		return
 	}
 
+	// Determine if the proposed block has a sane, non-nil proof-of-lock.
+	if cs.Proposal.POLRound >= 0 && cs.Proposal.POLRound < cs.Round {
```

Comment:
Is it a conscious choice to not fix the blind prevote bug in `if cs.LockedBlock != nil {` (similar to the PBTS [PR](https://github.com/tendermint/tendermint/pull/6986/files))? Are we only adding this new case when `LockedBlock == nil` to

Reply:
Update: fixed, due to `LockedBlock` logic fix above